pdata/ptrace/traces.go

```diff
@@ -70,14 +70,34 @@ func (ms Traces) ResourceSpans() ResourceSpansSlice {
 	return newResourceSpansSlice(&ms.getOrig().ResourceSpans)
 }
 
-// TraceState is a string representing the tracestate in w3c-trace-context format: https://www.w3.org/TR/trace-context/#tracestate-header
+// Deprecated: [v0.60.0] use pcommon.TraceState.
 type TraceState string
 
 const (
-	// TraceStateEmpty represents the empty TraceState.
+	// Deprecated: [v0.60.0] use pcommon.TraceState.
```

Comment:
People using this are using a specific instance of the trace state. Perhaps the godoc tell people how to accomplish the same with the new API?

Reply:
Clarified how to use it temporary. There are only 2 usages of this in contrib, will take as an action item to provide an API or a helper to check for empty.